checkita-core/src/main/resources/db/specific/oracle/V1.1__create_job_state.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE "${defaultSchema}"."results_job_config"
+(
+    "job_id"            TEXT             NOT NULL,
```

Comment:
In Oracle db change type for `job_id` to `VARCHAR(512)` and for `config` to `CLOB`